api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java

```diff
@@ -154,4 +164,26 @@ public void testDeleteLocationToOrgUnitMappingsByLocation() {
 				Context.getLocationService().getDefaultLocation());
 		Assert.assertEquals(0, Context.getService(DHISConnectorService.class).getAllLocationToOrgUnitMappings().size());
 	}
+
+	@Test
+	public void testDHISMappingExport() throws IOException {
+		DHISConnectorService dhisConnectorService = Context.getService(DHISConnectorService.class);
+		ReportDefinitionService reportDefinitionService = Context.getService(ReportDefinitionService.class);
+
+		DHISMapping mapping = new DHISMapping();
+		mapping.setName("mapping-test-unit");
+		mapping.setCreated(new Date().getTime());
+		PeriodIndicatorReportDefinition pir = new PeriodIndicatorReportDefinition();
```

Comment:
How about replacing `pir` with `reportDefinition` or something?

Reply:
Sure @jayasanka-sack On it